### ISSUE TITLE
Missing warnings for break and continue statements outside of loops

### ISSUE THREAD

**Issue by correctmost** [astral-sh/ty#1026]
### Summary

ty does not seem to warn about `break` and `continue` statements outside of loops:

```python
break
continue
```

--> https://play.ty.dev/cc42a4fb-75c1-446a-8c0c-c02af2c0df1a

Pyright, Pyrefly, mypy, and ruff issue warnings for both statements.

### Version

ty 0.0.1-alpha.18

**Comment by AlexWaygood** [comment]
Thanks for the report! This is covered by https://github.com/astral-sh/ruff/issues/17412 (we'd almost certainly implement the syntax error for Ruff and ty simultaneously -- most of our syntax error detection is shared between the two projects)
